### PR TITLE
MNIST Convolve

### PR DESCRIPTION
**Pull request by aidanhor-n**

### CODE REVIEW

File: images/pole-balancing-solution [hunk 1]
```diff
@@ -1,12 +1,8 @@
 digraph {
 	node [fontsize=9 height=0.2 shape=circle width=0.2]
-	0 -> 4 [color=green penwidth=0.19137870371341706 style=solid]
-	1 -> 4 [color=green penwidth=0.682824170589447 style=solid]
-	2 -> 4 [color=green penwidth=0.5564530551433563 style=solid]
-	3 -> 4 [color=green penwidth=0.741154944896698 style=solid]
-	5 -> 4 [color=green penwidth=0.11149876136332751 style=solid]
-	3 -> 6 [color=green penwidth=0.4187172412872314 style=solid]
-	6 -> 4 [color=green penwidth=0.12155715227127076 style=solid]
```
Comment (GregoryConrad) [inline images/pole-balancing-solution:9]: plz revert this to the one in master

File: mnist.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import logging
+
+import neat.population as pop
+import neat.experiments.MNIST.config as c
```
Comment (GregoryConrad) [inline mnist.py:4]: put this in the neats/experiments/MNIST dir

File: neat/population.py [hunk 3]
```diff
@@ -27,6 +27,7 @@ def __init__(self, config):
 
     def run(self):
         for generation in range(1, self.Config.NUMBER_OF_GENERATIONS):
+            print(f"GEN {generation} of {self.Config.NUMBER_OF_GENERATIONS}")
```
Comment (GregoryConrad) [inline neat/population.py:30]: plz revert this file to one in master

File: neat/utils.py [hunk 4]
```diff
@@ -62,8 +62,10 @@ def create_prediction_map(genomes, dataset, config):
         phenotype.to(config.DEVICE)
         for input in dataset:
             input.to(config.DEVICE)
+            #print(input.shape)
```
Comment (GregoryConrad) [inline neat/utils.py:65]: remove this line

File: neat/utils.py [hunk 5]
```diff
@@ -62,8 +62,10 @@ def create_prediction_map(genomes, dataset, config):
...
             prediction = phenotype(input)
-            results.append(prediction.numpy())
+            print(prediction)
```
Comment (GregoryConrad) [inline neat/utils.py:67]: remove this line

**Review by GregoryConrad — CHANGES_REQUESTED** [review]

**Review by aidanhor-n — COMMENTED** [review]
all changed reviewed

File: neat/population.py [hunk 6]
```diff
@@ -41,7 +41,7 @@ def run(self):
                 )
 
             best_genome = utils.get_best_genome(self.population)
-
+
```
Comment (GregoryConrad) [inline neat/population.py:44]: can you run format on this file plz or revert it back to master

**Review by GregoryConrad — CHANGES_REQUESTED** [review]
please also move pole run back

**Review by GregoryConrad — APPROVED** [review]

**Pull request merged**